### PR TITLE
Move dg-docassembly to kustomize

### PR DESCRIPTION
**Pull request by damongreen123**
https://tools.hmcts.net/jira/browse/DTSSE-15

Migrate dg-docassembly to kustomize config in flux

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ x] No
```

### CODE REVIEW

**Review by linusnorton — DISMISSED** [review]

File: k8s/namespaces/dg/dg-docassembly/prod.yaml [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+metadata:
+  name: dg-docassembly
+spec:
+  values:
+    java:
+      environment:
+        DOCMOSIS_TEMPLATES_ENDPOINT: "https://docmosis.platform.hmcts.net"
+        DOCMOSIS_CONVERT_ENDPOINT: "https://docmosis.platform.hmcts.net/rs/convert"
+        DOCMOSIS_RENDER_ENDPOINT: "https://docmosis.platform.hmcts.net/rs/render"
+        ENABLE_FORM_DEFINITION_ENDPOINT: "false"
```
Comment (yogesh-hullatti) [inline k8s/namespaces/dg/dg-docassembly/prod.yaml:12]: We don't need this line as this is same as in the https://github.com/hmcts/dg-docassembly-api/blob/f4751578feadc6da0936c7e06001234e4a2bd58b/charts/dg-docassembly/values.yaml#L43

File: k8s/namespaces/dg/dg-docassembly/prod.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+        ENABLE_TEMPLATE_RENDITION_ENDPOINT: "true"
```
Comment (yogesh-hullatti) [inline k8s/namespaces/dg/dg-docassembly/prod.yaml:13]: We don't need this line as this is same as in the https://github.com/hmcts/dg-docassembly-api/blob/f4751578feadc6da0936c7e06001234e4a2bd58b/charts/dg-docassembly/values.yaml#L44

File: k8s/prod/common/dg/dg-docassembly.yaml [hunk 3]
```diff
@@ -1,40 +0,0 @@
...
-    repository: https://hmctspublic.azurecr.io/helm/v1/repo/
-    name: dg-docassembly
-    version: 1.1.0
-  values:
-    java:
-      replicas: 2
-      memoryRequests: "512Mi"
-      cpuRequests: "250m"
-      memoryLimits: "2048Mi"
-      cpuLimits: "1500m"
```
Comment (yogesh-hullatti) [inline k8s/prod/common/dg/dg-docassembly.yaml:25]: Do we need this cpuLimits? As default value is 1000m. https://github.com/hmcts/chart-java

File: k8s/namespaces/dg/dg-docassembly/prod.yaml (hunk 1 above)
Comment (damongreen123) [reply k8s/namespaces/dg/dg-docassembly/prod.yaml]: removed

File: k8s/prod/common/dg/dg-docassembly.yaml (hunk 3 above)
Comment (damongreen123) [reply k8s/prod/common/dg/dg-docassembly.yaml:25]: Added to deafult.

**Review by linusnorton — APPROVED** [review]

File: k8s/namespaces/dg/dg-docassembly/dg-docassembly.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  releaseName: dg-docassembly
+  chart:
+    git: git@github.com:hmcts/hmcts-charts
+    ref: master
+    path: stable/dg-docassembly
+  values:
+    java:
+      useInterpodAntiAffinity: true
+      replicas: 2
+      memoryRequests: "512Mi"
```
Comment (adusumillipraveen) [inline k8s/namespaces/dg/dg-docassembly/dg-docassembly.yaml:15]: Do u need high values for all envs? or just aat/prod ?

**Review by adusumillipraveen — COMMENTED** [review]
Minor comments, good otherwise.

File: k8s/namespaces/dg/dg-docassembly/dg-docassembly.yaml (hunk 4 above)
Comment (damongreen123) [reply k8s/namespaces/dg/dg-docassembly/dg-docassembly.yaml:15]: I can limit demo and ITHC to 256 if you prefer? @adusumillipraveen

Comment (adusumillipraveen) [reply k8s/namespaces/dg/dg-docassembly/dg-docassembly.yaml:15]: yeah, just move this to environments where you want to override like aat, prod.  Otheriwise, everything looks good .

**Review by adusumillipraveen — APPROVED** [review]

**Pull request merged**